packages/lit-dev-content/site/tutorials/content/async-directive/00.md

```diff
@@ -0,0 +1,28 @@
+In this tutorial, we'll take a look at creating custom async directives, which
+are a powerful feature in Lit for building template helpers.
+
+Async directives provide additional API and lifecycle to the standard directive
+API, allowing the directive to update its value asynchronously and cleanup any
+subscriptions when the directive is no longer in use.
+
+<h2>What you'll learn</h2>
+
+* How to make a basic `Directive`
+* The additional directive API added by `AsyncDirective`
+* How to update a directive's value asynchronously using `setValue`
+* How to perform cleanup work in the `disconnected()` callback
+* Why implementing the `reconnected()` callback is also important
+
+<h2>What you'll build</h2>
+
+You will make a `timeAgo(date)` directive that accepts a JavaScript `Date`, and
+renders human-readable string showing the time that has passed. The directive
+will re-render its value periodically, so that the string always reflects the
+time that has passed. We'll be able to then use the directive in a variety of
+scenarios, as shown to the right.
```

Comment:
Fixed icon change.

Reply:
I actually don't think the date picker example is that great, because once the elapsed time is more than like a minute, you can't really observe that it's actually changing async (because it will just report the same course-grained elapsed time for a long time, and just look static). 

"Try changing the date picker" doesn't really emphasize that the elapsed time will stay accurate as time passes, so not sure it makes sense to dwell on it. Actually seeing the time tick up (while maybe not the best UX -- the end suggests setting the frequency to compensate for this) -- is what drives home that it's async. Not sure if that argues for removing the date picker all together, but seems fine to leave it.